Documentation/contributing/development/hubble.rst

```diff
@@ -0,0 +1,38 @@
+.. only:: not (epub or latex or html)
```

Comment:
should this belong to hubble repo ? I can see one benefit having it here is to use the same website :thinking:

Reply:
Fair question. All the other Hubble documentation has moved to the Cilium repo: https://github.com/cilium/hubble/tree/master/Documentation.

Reply:
Hubble documentation is now consolidated here as Hubble is very tightly coupled with Cilium. If we start adding back documentation to the Hubble repo it won't be clear where to look for what. So I think here is where we want this doc. What could be done though is to add a link to this page in Hubble's [CONTRIBUTING.md](https://github.com/cilium/hubble/blob/master/CONTRIBUTING.md) file.

Reply:
oh right, I was checking this one only https://github.com/cilium/hubble/blob/master/CONTRIBUTING.md.

Thanks :100: